src/content/docs/tutorial-errors/solve-critical-errors.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Solve your critical errors
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+import apmErrorsAttributes from 'images/apm_screenshot-crop_errors-details.webp'
+
+import apmLogs from 'images/apm_screenshot-crop_errors-logs.webp'
+
+import apmLogsZoom from 'images/apm_screenshot-crop_errors-logs-zoomed.webp'
+
+import apmStackTraces from 'images/apm_screenshot-crop_errors-stack-traces.webp'
+
+import apmErrorIDE from 'images/apm_screenshot-crop_errors-IDE.webp'
+
+Once you've chosen your error group, New Relic takes you to a new page that displays attributes, logs, and traces about the error. This part of the tutorial guides you through two recommended avenues for analyzing an error, your logs and stack traces. 
+
+## Objectives [#objectives]
+
+This doc builds off the problem scoped from the [Respond to service outages](/docs/tutorial-errors/respond-outages) doc of this tutorial series. In this doc, you'll:
+
+* Locate the error code associated with a failing API with logs
+* Use stack traces to find where in your code the error is occurring
+
+## Locate the source of an error with logs and stack traces [#analyze]
+
+The errors summary tells the story behind your error groups. Depending on how you've configured New Relic, each service might display different kinds of information about an error. For example, disabling distributed tracing limits trace details about external services.
+
+<Steps>
+    <Step>
+
+    ### Open the Log details window
+
+Looking at our example, the APM agent has pulled logs related to `api-gateway`, giving us event data about our error:
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmLogs}
+    />
+
+Our logs in context capability formats your logs information, though you have the option to investigate the unformatted log as well. 
+    </Step>
+    <Step>
+
+    ### Find the failure point in your system
+        
+Depending on the nature of the log, you may have more or less logging detail to sort through. Because you spent time upfront narrowing down the service closest to the failure and choosing the likely error group, you have time to read your logs. 
```

Comment:
I found this ¶ a bit confused conceptually.

* Why do I care that I have more or less detail?
* What is the time pressure I'm under?

I think it can be more direct and instructive.